src/core/syscalls/business_logic_syscall_handler.rs

```diff
@@ -329,6 +395,61 @@ mod tests {
     }
 
     #[test]
+    fn get_tx_info_test() {
+        // create data and variables to execute hint
+
+        let mut vm = vm!();
+        add_segments!(vm, 3);
+        //println!("vm fp: {:?}");
+
+        // insert syscall_ptr
+        let syscall_ptr = Relocatable::from((2, 0));
+
+        let version = BigInt::from_str("1").unwrap();
+        let account_contract_address = BigInt::from_str("1").unwrap();
+        let max_fee = BigInt::from_str("2").unwrap();
+        let signature_len = BigInt::from_str("1").unwrap();
+        let signature = Relocatable::from((3, 0));
+        let transaction_hash = BigInt::from_str("1").unwrap();
+        let chain_id = BigInt::from_str("1").unwrap();
+        let nonce = BigInt::from_str("1").unwrap();
+
+        memory_insert!(
+            vm,
+            [
+                ((1, 0), syscall_ptr),
+                ((2, 0), version),
+                ((2, 1), account_contract_address),
+                ((2, 2), max_fee),
+                ((2, 3), signature_len),
+                ((2, 4), signature),
+                ((2, 5), transaction_hash),
+                ((2, 6), chain_id),
+                ((2, 7), nonce)
+            ]
+        );
+
+        // syscall_ptr
+        let ids_data = ids_data!["syscall_ptr"];
+
+        let hint_data = HintProcessorData::new_default(GET_TX_INFO.to_string(), ids_data);
+        // invoke syscall
+        let syscall_handler = SyscallHintProcessor::new_empty().unwrap();
+        let err = syscall_handler.execute_hint(
+            &mut vm,
+            &mut ExecutionScopes::new(),
+            &any_box!(hint_data),
+            &HashMap::new(),
+        );
+
+        assert_eq!(
+            err,
+            Err(VirtualMachineError::UnknownHint(
+                "Hint not implemented".to_string()
```

Comment:
shouldn't the result of the hint execution be successful?  why is returning error?

Reply:
Because the response is not implemented yet, so i return a `SyscallHandlerError::NotImplemented` which the syscall handler `execute_hint` always maps to a Unknown hint on error

Reply:
good, thanks for explaining it